lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@smaeda-ks If that's the case i think i'm facing the same issue, there is something wrong with the functionality. I will create a ticket to investigate it further. 
Thanks for pointing this out.